profile/README.md

```diff
@@ -0,0 +1,22 @@
+<p align="center">
+  <a href="https://www.nillion.com">
+    <picture>
+    <img alt="nillion logo" src="../assets/nillion.png" width="auto" height="100">
+    </picture>
+  </a>
+</p>
+
+</br>
+<h1 align="center">Welcome to the Nillion Network</h1>
+
+
+Our goal is to build humanity's first blind computer. Nillion is a secure
+computation network that decentralizes trust for high value data in the same way
+that blockchains decentralized transactions.
+
+Get started with Nillion in one of the following links:
+* <a href="https://nillion.com">Website</a>
+* <a href="https://docs.nillion.com">Documentation</a>
+* <a href="https://twitter.com/nillionnetwork">X (Twitter)</a>
+* <a href="https://warpcast.com/nillion">Warpcast</a>
+* <a href="https://www.linkedin.com/company/nillionnetwork/mycompany">LinkedIn</a>
```

Comment:
Shall we add discord and telegram links?

Reply:
done